scripts/machineconfig/compile_butane.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
nit: What's your opinion on another file name like "create_machine_config.sh" with the purpose and not how it achieves that?